tests/integration/shell/key.py

```diff
@@ -141,6 +172,34 @@ def test_list_acc(self):
         expect = ['Accepted Keys:', 'minion', 'sub_minion']
         self.assertEqual(data, expect)
 
+    def test_list_acc_eauth(self):
+        '''
+        test salt-key -l with eauth
+        '''
+        self._add_user()
+        data = self.run_key('-l acc --eauth pam --username {0} --password {1}'.format(USERA, USERA_PWD))
+        expect = ['Accepted Keys:', 'minion', 'sub_minion']
+        self.assertEqual(data, expect)
+        self._remove_user()
+
+    def test_list_acc_eauth_bad_creds(self):
+        '''
+        test salt-key -l with eauth and bad creds
+        '''
+        self._add_user()
+        data = self.run_key('-l acc --eauth pam --username {0} --password wrongpassword'.format(USERA))
+        expect = []
```

Comment:
Not sure if this is actually a bug since the command is returning nothing. I guess I would expect it to return some error about an auth failure. ping @DmitryKuzmenko maybe you know if this is expected behavior with this test or a bug since you wrote this feature